issue_01/script.js

```diff
@@ -0,0 +1,4 @@
+const ul = document.getElementsByTagName('ul');
+const li = document.createElement("li");
+li.textContent = "これです"
+ul.item(0).appendChild(li);
```

Comment:
IMO
If you get the id instead of `getElementsByTagName`, you can also `appendChild` without specifying the index. Please try it.